packages/superset-ui-legacy-plugin-chart-map-box/src/utils/geo.js

```diff
@@ -8,7 +7,7 @@ export function kmToPixels(kilometers, latitude, zoomLevel) {
   // Algorithm from: http://wiki.openstreetmap.org/wiki/Zoom_levels
   const latitudeRad = latitude * (Math.PI / 180);
   // Seems like the zoomLevel is off by one
-  const kmPerPixel = (EARTH_CIRCUMFERENCE_KM * Math.cos(latitudeRad)) / Math.pow(2, zoomLevel + 9);
+  const kmPerPixel = (EARTH_CIRCUMFERENCE_KM * Math.cos(latitudeRad)) / 2 ** (zoomLevel + 9);
```

Comment:
```suggestion
  const kmPerPixel = (EARTH_CIRCUMFERENCE_KM * Math.cos(latitudeRad)) / (2 ** (zoomLevel + 9));
```
Again totally optional, but I wonder if parens make this easier to read.

Reply:
I think prettier removes them so would have to `prettier-ignore`